docs/datasync/soft-delete.md

```diff
@@ -0,0 +1,233 @@
+---
+id: soft-delete
+title: Soft Deletes with delta queries
+sidebar_label: Setting up Soft Deletion
+---
+
+This is the simplest strategy for Data Synchronization that is also quite straightforward to set up.
+
+## Setup
+
+Start off with the official Graphback template for MongoDB [*here*](https://GitHub.com/aerogear/graphback/tree/master/templates/ts-apollo-mongodb-backend) to follow along. Adding this strategy is relatively simple, consisting of the following two steps:
+
+### Annotate the required models
+
+Add the `@datasync` annotation to your model(s) in your GraphQL SDL found in the `model` folder:
+
+```graphql
+""" 
+@model
+@datasync
+"""
+type Comment {
+  id: ID!
+  text: String
+  description: String
+}
+```
+
+The `@datasync` annotation ensures consistency of your data and gives you delta queries.
+
+> **NOTE**: The `@datasync` annotation is a shorthand for two annotations `@versioned` and `@delta`. Refer to the last section for details.
+
+`@datasync` transforms your model by adding two fields: `updatedAt` and `createdAt`
+
+```graphql
+""" 
+@model
+@datasync 
+"""
+type Comment {
+  id: ID!
+  text: String
+  description: String
+  createdAt: String
+  updatedAt: String
+}
+```
+
+The `@datasync` annotation adds a `sync` query or a delta query:
+```graphql
+type Query {
+  syncComments(lastSync: String!, filter: CommentFilter): CommentDeltaList!
+}
+```
+
+This allows you to get all the changed documents(updated and deleted) in a collection since the `lastSync` timestamp. Internally this uses the `updatedAt` database field to check if any documents in the database have been modified, by comparing client provided `lastSync` timestamp value.
```

Comment:
```suggestion
This allows you to get all the changed documents (updated and deleted) in a collection since the `lastSync` timestamp. Internally this uses the `updatedAt` database field to check if any documents in the database have been modified, by comparing client provided `lastSync` timestamp value.
```